engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -660,6 +729,37 @@ public JSONObject getPorts(@PathParam("operatorName") String operatorName)
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/ports")
```

Comment:
@tweise Are you suggesting to use /operatorName/ports for listing module ports? In this can I can handle it the same way as properties.